types/core-js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for core-js 0.9
+// Type definitions for core-js 0.10
```

Comment:
It looks like `core-js 0.10` was never released and it went straight from `0.9` to `1.0`. You don't *have* to update the header if the library version didn't change.

Reply:
The core-js project is up past 2.0 now. I thought this was the version number for the typing. Should I revert this? I am not claiming that my changes bring this typing in line with a particular version of core-js. I am just adding a couple missing function definitions.

Reply:
Yeah, change it back then.